codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -156,14 +175,10 @@ public void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream,
         }
         // Save the context. We'll use this later when we write pending bytes.
         this.ctx = ctx;
-        try {
-            FlowState state = state(stream);
-            state.newFrame(payload);
-            state.writeBytes(state.writableWindow());
-            flush();
-        } catch (Throwable e) {
-            payload.error(e);
-        }
+        FlowState state = state(stream);
+        state.newFrame(payload);
+        state.writeBytes(state.writableWindow());
+        flush();
```

Comment:
@Scottmitch @nmittler I removed the try catch to avoid calling `error()` twice in case `error()` throws an Exception. I think this should be absolutely fine.